src/core/interval.js

```diff
@@ -40,7 +42,9 @@ export type GraphIntervalPartition = $ReadOnlyArray<GraphInterval>;
  */
 export function partitionGraph(graph: Graph): GraphIntervalPartition {
   const nodes = Array.from(graph.nodes());
-  const timefulNodes = nodes.filter((x) => x.timestampMs != null);
+  const timefulNodes: $ReadOnlyArray<TimefulNode> = (nodes.filter(
+    (x) => x.timestampMs != null
+  ): any);
```

Comment:
This one was tricky, as the base type of `Node[]` has a `timestampMs: number | null`. And null isn't supported for comparisons.

We're aware of this, which is why we're doing this `nodes.filter()`... but flow doesn't automatically enrich the type here to not have a non-null `timestampMs`.